clouddriver-web/src/main/groovy/com/netflix/spinnaker/clouddriver/config/ThreadConfiguration.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2020 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.netflix.spinnaker.clouddriver.config;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.scheduling.concurrent.ThreadPoolTaskScheduler;
+
+@Configuration
+public class ThreadConfiguration {
```

Comment:
Seems that this class is not needed anymore.

Reply:
That one changes the pool number of threads that is used by anything that uses the @Scheduled notation.
That class allows to run the healthChecks of others (like SQL) while the accounts healthCheck also run.
Snap issue is the accounts healthCheck takes a lot of time to finish, and clouddriver never gets marked as running because it's waiting on the SQL healthCheck.